src/main/java/Data/Configuration.java

```diff
@@ -44,6 +44,9 @@ public static Properties getMerchantDetails() {
 		props.setProperty("clientId", "");
 		props.setProperty("clientSecret", "");
 
+		//PEM Key file path for decoding JWE Response Enter the folder path where the .pem file is located.
```

Comment:
in comment add the line "It is optional property, require adding only during JWE decryption."